cmd/doracled/cmd/get_oracle_key.go

```diff
@@ -100,8 +101,20 @@ func getOraclePrivKey(conf *config.Config, oracleRegistration *oracletypes.Oracl
 			return errors.New("the oracle private key already exists")
 		}
 
-		// else, get encryptedOraclePrivKey from Panacea and decrypt and SealToFile it
-		oraclePrivKey, err := crypto.Decrypt(nodePrivKey, oracleRegistration.EncryptedOraclePrivKey)
+		oraclePubKeyPath := conf.AbsOraclePubKeyPath()
+		unsealOraclePubKey, err := sgx.UnsealFromFile(oraclePubKeyPath)
+		if err != nil {
+			return fmt.Errorf("failed to unseal from file: %w", err)
+		}
+
+		oraclePubKey, err := btcec.ParsePubKey(unsealOraclePubKey, btcec.S256())
+		if err != nil {
+			return fmt.Errorf("failed to parse oraclePubKey: %w", err)
+		}
```

Comment:
As you can probably see the `conf.AbsOraclePubKeyPath()`, the oracle pubkey file(default name of `oracle_pub_key.json`) is used to store oracle public key and its remote report in `gen-oracle-key` cmd.

In general, the oracle to register probably don't have the file, thus how about to get the oracle public key from Panacea using query client?

Reply:
Ah yes, I understand. The `oracle_pub_key.json` will be generated when using `gen-oracle-key` cmd. So I think you're right. I'll change it.

Reply:
Fixed in [bd2279e](https://github.com/medibloc/panacea-doracle/pull/54/commits/bd2279e57d2a4307de9ce396325a2a35d9d43e16).